app/javascript/listings/__tests__/SingleListing.test.jsx

```diff
@@ -24,6 +27,12 @@ const listing = {
 };
 
 describe('<SingleListing />', () => {
+  beforeEach(() => {
+    window.timestampToLocalDateTimeLong = () =>
```

Comment:
I couldn't define common mock directory (for javascript) to put this mock function

Reply:
Also I'm making this function to return string without any actual logic here in the mock since the test can fail if the user has different locale settings in their system.